src/lib/migration_sle/repos_workflow.rb

```diff
@@ -0,0 +1,67 @@
+# ------------------------------------------------------------------------------
```

Comment:
NP: This copy header is different to the common one we are using. See https://github.com/yast/yast-migration-sle/pull/11/files#diff-5c3088bdecc3e77627fb22bd8054d4418b7affd976db2bc88925783dcf5dcadbR1-R19